clients/drcachesim/simulator/caching_device.h

```diff
@@ -61,13 +61,17 @@ namespace drmemtrace {
 class snoop_filter_t;
 class prefetcher_t;
 
+// NINE = Non-Inclusive Non-Exclusive.
+enum class cache_inclusion_policy_t { NINE, INCLUSIVE, EXCLUSIVE };
```

Comment:
Hmm, is "NINE" a widely recognized acronym?  Would it be better to be named "NEITHER"?  That seems more understandable at a glance without having to go hunt down this header comment.

Reply:
Honestly, I had the same question, but I found that label used on Wikipedia's entry for inclusion policy, as well as in unrelated online materials I found (such as https://youtu.be/zJzxHxThgPg?si=Z_vEWVl9zHOYstfI&t=188).  So it's an established label even if not universally known.  I don't like NEITHER since it implies only two other options, and there could be other variations added later, for example to model some real-world caches that are just _mostly_ exclusive.  The alternative I considered is NON_INC_NON_EXC, which is maybe more obvious to those unfamiliar with NINE, though I obviously favor keeping the "official" name for this policy.

Reply:
The problem is NINE looks like 9.  N_I_N_E would be better IMHO.  I would vote for NON_INC_NON_EXC.